src/Radicle/Internal/PrimFns.hs

```diff
@@ -75,24 +75,26 @@ purePrimFns = fromList $ allDocs $
           [] -> pure $ bindingsToRadicle (pureEnv :: Bindings (PrimFns m))
           xs -> throwErrorHere $ WrongNumberOfArgs "pure-env" 0 (length xs)
       )
-    , ("apply"
+    , ( "apply"
       , "Calls the first argument (a function) using as arguments the\
         \ elements of the the second argument (a list)."
       , \case
           [fn, List args] -> callFn fn args
           [_, v]          -> throwErrorHere $ TypeError "apply" 1 TList v
           xs -> throwErrorHere $ WrongNumberOfArgs "apply" 2 (length xs))
-    , ( "read"
+    , ( "read-annotated"
       , "Parses a string into a radicle value. Does not evaluate the value."
-      , oneArg "read" $ \case
-          String s -> readValue s
-          v -> throwErrorHere $ TypeError "read" 0 TString v
+      , twoArg "read-annotated" $ \case
```

Comment:
Could you update the doc string so that it mentions the second argument? Same for `read-many-annotated`.